porcupine/plugins/run/dialog.py

```diff
@@ -105,8 +105,16 @@ def __init__(self, file_path: Path, project_path: Path, suggestions: list[histor
             validated_callback=self.update_run_button,
         )
 
+        ttk.Label(content_frame, text="Substitutions:").pack(anchor="w")
+
         sub_text = "\n".join("{%s} = %s" % pair for pair in substitutions.items())
-        ttk.Label(content_frame, text=f"Substitutions:\n{sub_text}\n").pack(fill="x")
+        sub_textbox = textutils.create_passive_text_widget(
+            content_frame, height=len(substitutions), width=1
```

Comment:
Currently it is always 7 lines tall, but wraps the text into more than 7 lines when you have long file paths or the dialog is very narrow. This makes some lines at the end invisible. We can disable wrapping to fix this:

```suggestion
            content_frame, height=len(substitutions), width=1, wrap="none"
```